api/src/pcapi/scripts/move_offer/move_batch_offer.py

```diff
@@ -150,9 +150,30 @@ def _move_collective_offer_playlist(
 
 
 def _move_price_category_label(origin_venue: offerers_models.Venue, destination_venue: offerers_models.Venue) -> None:
-    db.session.query(offer_models.PriceCategoryLabel).filter(
-        offer_models.PriceCategoryLabel.venueId == origin_venue.id
-    ).update({"venueId": destination_venue.id}, synchronize_session=False)
+    price_category_labels = (
```

Comment:
[détail] pour encore plus de compréhension, ici j'aurais nommé ça 
```suggestion
    origin_price_category_labels = (
 ```